app/models/user.rb

```diff
@@ -26,6 +26,8 @@ class User < Common::RedisStore
   attribute :zip
   attribute :ssn
   attribute :loa
+  attribute :multifactor
+  attribute :authn_context
```

Comment:
You've got `authn_context` on both session and user models. Intentional?

Reply:
No, originally I went the route of putting it on session until I realized that the F/E would need it too for some weird logic they have going on there. I'll remove from session. Good catch.

Reply:
removed in recent push.